nautobot/core/templates/generic/object_list.html

```diff
@@ -111,7 +111,11 @@
                 <span class="caret"></span>
                 <span class="sr-only">Toggle Dropdown</span>
             </button>
-        {% else %}
+        {% elif permissions.add and 'import' in action_buttons %}
+            <button type="button" id="actions-dropdown" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+                Actions <span class="caret"></span>
+            </button>
+        {% elif 'export' in action_buttons %}
```

Comment:
Duplicated code here because Django templates don't support parenthesis in complex conditionals and I think this is preferred over expecting all maintainers have memorized Python's operator precedence